README.md

```diff
@@ -212,7 +234,8 @@ var config = {
     //   headers: {
     //     headerName: headerValue
     //   },
-    //   data: postBodyData
+    //   data: postBodyData,
+    //   withCredentials: true|false,
```

Comment:
nit: Maybe throw a line to explain why they might need `withCredentials` set to false or true.

Reply:
If they are providing the httpRequest agent, they just need to honor the setting. It is a common flag for ajax libraries.